apm-lambda-extension/Makefile

```diff
@@ -21,20 +21,16 @@ export AWS_FOLDER GOARCH ARCHITECTURE DOCKER_IMAGE_NAME DOCKER_REGISTRY
 .PHONY: all
 all: build
 
-check-licenses:
-	go install github.com/elastic/go-licenser@v0.4.0
-	go run github.com/elastic/go-licenser@v0.4.0 -d .
-
-update-licenses:
-	go install github.com/elastic/go-licenser@v0.4.0
-	go run github.com/elastic/go-licenser@v0.4.0 .
```

Comment:
Why are you replacing the license header generation? Afaics this is not covered by the notice generator (the `go-licenser` is the tool to add the license header to every code file).